### PR TITLE
Travis: Execute acceptance tests on dockerized RMQ.

### PR DESCRIPTION
**Pull request by cyrilgdn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,13 +28,18 @@ script:
 - make test
 - make vet
 - make website-test
+- docker-compose -f $(pwd)/scripts/docker-compose.yml up -d
+- $(pwd)/scripts/wait-rabbitmq-docker.sh $(pwd)/scripts/docker-compose.yml
```
Comment (multani) [inline .travis.yml]: You are passing the docker-compose file but it's not really used inside the script, I guess you can remove it?

File: scripts/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+version: "3"
+
+services:
+    rabbitmq:
+        image: rabbitmq:${RABBITMQ_VERSION:-3.8}-management-alpine
+        ports:
+            - 15672:15672
```
Comment (multani) [inline scripts/docker-compose.yml:10]: To make things more explicit, could you also pass the environment variables that you defined for the username, password in here?

```
env:
  RABBITMQ_DEFAULT_USER: ${RABBITMQ_USERNAME:guest}
  RABBITMQ_DEFAULT_PASS: ${RABBITMQ_PASSWORD:guest}
```

File: scripts/wait-rabbitmq-docker.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+TIMEOUT=30
+
+if [ ! -e "$COMPOSE_FILE" ]; then
+    echo "Unable to find docker-compose file: $COMPOSE_FILE"
+    exit 1
+fi
+
+echo "Waiting for RabbitMQ to be up"
+i=0
+until curl -s http://localhost:15672/api > /dev/null; do
```
Comment (multani) [inline scripts/wait-rabbitmq-docker.sh]:
```suggestion
until curl -s "${RABBITMQ_ENDPOINT}/api" > /dev/null; do
```

**Review by multani — CHANGES_REQUESTED** [review]
Thanks @cyrilgdn for enabling this, it would show that the test I wrote in #45 (without the fix) would have failed so :+1: for that!

File: .travis.yml (hunk 1 above)
Comment (cyrilgdn) [reply .travis.yml]: Indeed :+1: , a bit copy/pasted from another provider where it was used :sweat_smile:

**Comment by cyrilgdn** [comment]
Thanks @multani  for your review !

I updated the PR based on your remarks

**Review by multani — APPROVED** [review]
Looks good to me, let me know when it's merged and I can rebase #45 on top of it :+1:

**Pull request merged**